### ISSUE TITLE
Missing parts of chapters

### ISSUE THREAD

**Issue by ppdewolf** [julienjamme/manual_tau_argus_to_quarto#2]
Splitting chapters into sections misses a few paragraphs. Sometimes a chapter starts with some text before the first subsection. That text seems to be missing after the split into sections? See e.g. teh chapter on "tour of tau-argus".

**Comment by julienjamme** [comment]
Indeed, I split the chapters into sections using the ## pattern. So, if some text or pictures are located before this pattern, the split misses them. I'll check that and make the adjustments manually.
